posydon/utils/x_ray_luminosities.py

```diff
@@ -0,0 +1,188 @@
+
+"""Determins XRB type and calculate X-ray luminosities."""
+
+
+__authors__ = [
+    "Devina Misra <devina.misra@unige.ch>",
+]
+
+
+import os
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import newton
+from scipy.integrate import quad
+from posydon.utils import constants as const
+import copy
+import warnings
+from scipy.interpolate import PchipInterpolator
+from posydon.utils.common_functions import eddington_limit
+
+
+PATH_TO_POSYDON = os.environ.get("PATH_TO_POSYDON")
+
+
+# def xrb_type(binary, idx=-1):
```

Comment:
Please explain, why here is code, which isn't aimed to be used. May consider to simply remove it.